exercises/practice/reverse-string/.docs/instructions.append.md

```diff
@@ -0,0 +1,13 @@
+# ignore
```

Comment:
Why the heading of ignore?  If it is a header, it should also be capitalized, I would think.

Reply:
markdown files have to have an H1, but it is ignored for append files. Just a quirk of exercism markdown standards.